### PR TITLE
Add 1.14 release tests for Windows on aks-engine

### PR DESCRIPTION
**Pull request by adelina-t**

### CODE REVIEW

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml [hunk 1]
```diff
@@ -76,6 +76,46 @@ periodics:
...
+    preset-azure-acsengine: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-master
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=release-1.14"
```
Comment (PatrickLang) [inline config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: Question for reviewers: will this branch receive updates daily?

**Comment by PatrickLang** [comment]
/LGTM

@BenTheElder or @spiffxp - can you approve? Is this the right way to get the 1.14 Windows test pass created for the release team to watch?

File: config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml (hunk 1 above)
Comment (spiffxp) [reply config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml]: Yes, @hoegaarden is responsible for this and post in #sig-release when they're done

**Comment by spiffxp** [comment]
/approve

>Is this the right way to get the 1.14 Windows test pass created for the release team to watch?

Let's let this sit and see what the signal looks like.  We can followup with a PR to release-informing if it makes sense

**Pull request merged**